au4/src/project/internal/audacityproject.h

```diff
@@ -77,6 +77,8 @@ class Audacity4Project : public IAudacityProject
 
     uintptr_t au3ProjectPtr() const override;
 
+    au::project::IProjectHistoryPtr projectHistory() const override;
```

Comment:
I think it would be better to take it out of Audacity4Project and make it like other similar services (TrackeditInteraction, SelectionController etc.).

Reply:
fixed, please have a look